server/resources/resource_management_test.go

```diff
@@ -1624,6 +1624,30 @@ var _ = Describe("Resource manager", func() {
 
 			})
 		})
+		Describe("Whether old data is not overridden in update_in_transaction", func() {
+			BeforeEach(func() {
+				javascriptCode := fmt.Sprintf(`
+					tx = context.transaction;
+					res = gohan_db_fetch(tx, "%s", "%s", "");
```

Comment:
Long live JS!
Sorry guys for that but that was easiest way for reproducing this issue in UT ;)

Reply:
I'd dispute what I'd wish for the JS exts, but definitely it'll not be a "long live" ;)

You could try to `RegisterEnvironment` with a `MockEnvironment` with `DoAndReturn(func() { yourGoCode() }` registered on `HandleEvent`